packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -202,14 +206,12 @@ Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_get = passOnSucces
     kind: "MockApiDefinition",
   },
   {
-    uri: "/subscriptions/:subscriptionId/resourceGroups/:resourceGroup/providers/Azure.ResourceManager.Resources/topLevelTrackedResources/:topLevelResourceName/providers/Azure.ResourceManager.Resources/extensionsResources/:extensionName",
+    uri: "/:resourceUri/providers/Azure.ResourceManager.Resources/extensionsResources/:extensionName",
```

Comment:
Does this work for four identical `uri`s?
Also, does it allow both single `/` and double `//`?

Reply:
Fixed by [9658831](https://github.com/Azure/typespec-azure/pull/2203/commits/9658831fac4fdc139b493a39e100fa115d01cc17)